wrims_v2/wrims_v2/src/test/test_heclib7/Test_heclib7.java

```
@@ -95,9 +117,25 @@ public void writeData(String dvPath){
 			}
 		}catch (Exception e) {
 			System.out.println("Could not open dv file. "+e);
-			return;
+			return null;
 		}
+		return outPath;
 	}
 	
+	public TimeSeriesContainer readData(String dvPath, String fullName){
+		HecDss dvDss;
+		DataContainer tsc;
+		try{
+			dvDss = HecDss.open(dvPath);
+			tsc = dvDss.get(fullName);
+			dvDss.close();
+			assert (tsc instanceof TimeSeriesContainer) ;
+			return (TimeSeriesContainer)tsc;
+		}
+		catch (Exception e) {
```

let the method throw the exception instead of returning null